Dockerfile

```diff
@@ -40,5 +38,12 @@ FROM base
 # Copy built application
 COPY --from=build /app /app
 
-# Start the server by default, this can be overwritten at runtime
-CMD [ "npm", "run", "start" ]
+# Set to `publish` or `api`
```

Comment:
What do you think about consistently only using `spark-api` or `api`? Currently it's mixed: On disk it's called `spark-api`, here the arg is `api`. What do you think about using `spark-api` everywhere, or just `api`? `spark-api` is the more explicit one, and matches the fly project name. `api` it's nicer, because there's less repetition (`spark-api/spark-api`)

Reply:
Great question!

FWIW, the Fly.io project is called `spark`, I think the name `spark-api` is already taken by somebody else. But you are right that all other Fly.io project names contain the `spark-` or `voyager-` prefixes.


I was thinking that maybe we can change the directory names to drop the prefix and rename `spark-api/spark-api` to `spark-api/api`, `spark-api/spark-publish` to `spark-api/publish`, and so on.

However, I also see how using the Fly project name as the directory name makes things easier to understand.

In that light, should we use the names `spark` (for API) and `spark-publish`?

The downside is that we have more to type at dev time, but I like how the script suffix matches the directory (sub-package) name.

```
npm run test:spark-publish
```

The remaining question is how to call the directory with the REST API server - should it be `spark-api` to match what it holds or `spark` to match the Fly project? I prefer `spark-api` as I find it less confusing.

Reply:
>  I think the name spark-api is already taken by somebody else

I don't think it is, or I haven't seen it. I called it `spark` in Fly because I assumed at that time it would be the only service we'd create for it :D I'm in favor of renaming it to `spark-api` in Fly.

I would prefer to go with `spark-api/{api,publish}`, I think we can remember that `spark-api/x` maps to `spark-x` on Fly. But the other option is fine for me too!

Reply:
We can also change this in the future, if we change our mind